integration-tests/tests/utils/utils.py

```
@@ -0,0 +1,113 @@
+    sort_bin = shutil.which("sort")
+    if jq_bin is None or sort_bin is None:
+        err_msg = "jq/sort executable not found"
+        raise RuntimeError(err_msg)
+
+    sorted_fp = NamedTemporaryFile(mode="w+", delete=True)  # noqa: SIM115
+    jq_proc = subprocess.Popen(
+        [jq_bin, "--sort-keys", "--compact-output", ".", str(json_fp)],
+        stdout=subprocess.PIPE,
+    )
+    try:
+        sort_proc = subprocess.run([sort_bin], stdin=jq_proc.stdout, stdout=sorted_fp, check=True)
```

Ruff (0.12.2)

Local variable `sort_proc` is assigned to but never used

Remove assignment to unused variable `sort_proc`

(F841)